mix.exs

```diff
@@ -60,7 +60,8 @@ defmodule Tesla.Mixfile do
       {:ibrowse, "~> 4.4.0", optional: true},
       {:hackney, "~> 1.6", optional: true},
       {:gun, "~> 1.3", optional: true},
-      {:castore, "~> 0.1", optional: true},
+      {:finch, "~> 0.3", optional: true},
+      {:castore, "~> 0.1", optional: true, override: true},
```

Comment:
why `override`?

Reply:
Oh yeah, right 👍  Bumped it to v0.1.5 now since that is what Finch requires.